client/test/grammars/snaps/keywords.bb

```diff
@@ -1,3 +1,73 @@
+# python keyword keyword.control
+if
+
+whatif
+
+ifnot
```

Comment:
DO you have an example .bb from a layer with python syntax that is not within a python() function?
If I understand right, that's what you are highlighting.

Reply:
ye right these test cases don't make sense if we don't expect Python code outside the Python function block. I will remove them or I can wrap all of them in Python function block.